### ISSUE TITLE
Accordion is missing the a visual indicator on focus

### ISSUE THREAD

**Issue by zipymonkey** [psu-libraries/psulib_base#269]
## Steps to Duplicate
- Go to https://psul-web.qa.k8s.libraries.psu.edu/guides/course
- Tab through the accordions
- You will not see any visual indicator that an accordion is focused on

## Expected Behavior
- Use browser default `:focus-visible` styles around the accordion header
